lib/Conversion/PTensorToLinalg/PTensorToLinalg.cpp

```diff
@@ -293,19 +295,63 @@ static BodyType buildTrivial(::mlir::Type typ) {
   };
 }
 
+// Builder for TOSA body.
+// The builder function takes an extra agrument ::mlir::Type
+// Many TOSA functions take two arguments and do the
+// same operation on both arguments.
+// TODO:
+//      1. Find a way to merge this function with trivialBuilder().
```

Comment:
Why would we need buildTrivial if we use TOSA?

Reply:
That's my thought too, should we just move to TOSA completely?

Reply:
When I investigated I realized that TOSA does not have unsigned types. I could not figure out if that's a gap or a design decision. If we can assume that TOSA can support unsigned ints then I the only reason for not using TOSA might be compile-time. But I tend to think that's probably acceptable at this point.